sharding/data_modeling.rst

```diff
@@ -10,6 +10,31 @@ Typically, you should choose that column as the distribution column which is the
 .. note::
   Citus also allows joining on non-distribution key columns by dynamically repartitioning the tables for the query. Still, joins on non-distribution keys require shuffling data across the cluster and therefore aren’t as efficient as joins on distribution keys.
 
-The best choice of distribution column varies depending on the use case and queries. Two common scenarios are the multi-tenant B2B application and the realtime analytics dashboard. In the former all tables include a tenant id and are distributed by it. When SQL queries are restricted to accessing data about a single tenant then Citus can execute them within a single shard. Having all data colocated in a shard is efficient and supports all SQL features. However running queries on a single shard limits the ability to parallelize execution.
+Use Cases and Tradeoffs
+-----------------------
 
-The other common option, realtime analytics, distributes by another column (such as user id). The queries in this scenario typically request aggregates information from multiple shards. This permits query parallelism but restricts some of the SQL features available, due to the constraints of being a distributed system.
+The best choice of distribution column varies depending on the use case and queries. Two common scenarios are the multi-tenant B2B application and the realtime analytics dashboard. Both cases involve tradeoffs.
+
+Multi-Tenant
+~~~~~~~~~~~~
+
+* ✔ SQL Coverage
```

Comment:
I like the simplicity of these, but it may not be immediately obvious what these points are (e.g. why SQL coverage is higher). I'd be OK to move these to a more text/paragraph format, and discuss these *after* the below discussion on multi-tenancy.